data/ka'het missions.txt

```diff
@@ -93,10 +93,11 @@ mission "First Contact: Ka'het: Remnant 1B"
 
 mission "First Contact: Ka'het: Quarg 1"
 	name "Abandoned Ringworld"
-	description "Visit a Quarg station to ask them about the abandoned ring in the middle of Ka'het space."
+	description "Visit the Quarg and ask them about the abandoned ring in the middle of Ka'het space."
 	landing
 	source "Querii'ka"
-	destination "Lagrange"
+	to complete
+		has "First Contact: Ka'het: Z Quarg 2: done"
```

Comment:
Looks like you have your Z in the wrong spot.
```suggestion
		has "First Contact: Z Ka'het: Quarg 2: done"
```

Reply:
Oopes.

Reply:
No, not oopes.  That puts it after the Quarg.

Reply:
I fixed that but I forgot to push the fix.  Github is one fix behind.

Reply:
Fixed.